### DISCUSSION TITLE
q-header don`t sticky

### DISCUSSION THREAD

**Discussion by JMHurtadoB** [quasarframework/quasar#14760] [General - Components / Directives / etc] [answered]
How make don´t sticky q-header in layout?

**Comment by MilosPaunovic** [comment]
Use `reveal` property.
https://codepen.io/milospaunovic/pen/oNybgdy?editors=1010

**Comment by rstoenescu** [comment, marked as answer]
Play with the "view" prop. Lowercase "h". https://quasar.dev/layout-builder
